addons/mass_mailing/tests/test_mailing_ui.py

```diff
@@ -79,3 +79,28 @@ def test_snippets_mailing_menu_toolbar_mobile_tour(self):
 
     def test_mass_mailing_code_view_tour(self):
         self.start_tour("/web?debug=tests", 'mass_mailing_code_view_tour', login="demo")
+
+
+class TestMassMailingActions(MassMailCommon):
+    def test_mailing_action_open(self):
+        mass_mailings = self.env['mailing.mailing'].create([
+            {'subject': 'First subject'},
+            {'subject': 'Second subject'}
+        ])
+        partners = self.env['res.partner'].search([], limit=2)
```

Comment:
Would rather explicitly use two specific partners to avoid using different partners every time, and to make sure it doesn't rely on demo/dirty db data

`partner_admin` and `partner_employee` should exist from `MailCommon`